src/js/fetches/movies.js

```diff
@@ -16,7 +17,28 @@ let page = 1;
 //     ? navigator.languages[0]
 //     : navigator.language;
 
-//  1.    ------ Function - fetch - Popular movies ------
+// 1.     ------ function fetch - get whole movies genres ------
+//?api_key=<<api_key>>&language=en-US
+
+export const getMovieGenres = async () => {
+  try {
+    const response = await fetch(searchGenresUrl + `?api_key=` + apiKey);
+    // response Status:404 handling
+    if (!response.ok) {
+      throw new Error(response.status);
+    }
+    const data = await response.json();
+    console.log('Poniżej przykladowy console.log dla listy gatunków');
+    console.log(data.genres);
+    setMovieGenresToStorage(data);
```

Comment:
jezeli to jest zapisywanie do LS, to raczej `saveMovieGenresToStorage` albo lepiej `Store.save(movieGenres)` lub `Store.saveMovieGenres(movieGenres)`.

Reply:
zmienione